init/database/sql/migrations/sql/6_add_documentation_toggle.go

```diff
@@ -0,0 +1,17 @@
+package migrations
+
+import "github.com/rubenv/sql-migrate"
+
+func AddDocumentationToggle() *migrate.Migration {
+	migrateToRSAKeys := migrate.Migration{
+		Id: "6",
+		Up: []string{`
+			INSERT INTO cqlw.gocms_settings (name, value, description) VALUES ('DOCUMENTATION', 'TRUE', 'Display documentation at /docs.  Boolean. ');
```

Comment:
Can't insert anything with vendor specific code. Take a look at how some of the other migrations work for settings and copy them.